.github/workflows/setup.yml

```
@@ -0,0 +1,56 @@
+name: Project setup
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - uses: ikalnytskyi/action-setup-postgres@v5
+        name: Set-up PostgreSQL
+        with: 
+          postgresql-version: 14
+          port: 5432
+          database: evershop
+          username: postgres
+          password: pgAdmin12345
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Setup database schema
+        run:  |
+          npm run build
+          npm run dev &
+        env:
+          NODE_ENV: production
+          DB_HOST: localhost
+          DB_PORT: 5432
+          DB_NAME: evershop
+          DB_USER: postgres
+          DB_PASSWORD: pgAdmin12345
+          ADMIN_EMAIL: example@gmail.com
+          ADMIN_PASSWORD: pSynidexxx.899
+          SUPPRESS_NO_CONFIG_WARNING: "true"
+
+      - name: Wait for development server
+        run: 
+            while ! curl -s -k http://localhost:3000 > /dev/null; do sleep 1; done
+
+      - name: Run e2e tests
+        working-directory: tests
+        run: |
+          npm install
+          npx playwright install chromium
+          npm run test:e2e
```

```suggestion
          npm run test:e2e path
```
Is path required or not?

---

Will update it.

---

Yes, the path is required. i will update it now.